ios/FluentUI.Demo/FluentUI.Demo/Demos/SideBarDemoController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class SideBarDemoController: DemoController {
+    override func loadView() {
+        view = UIView(frame: .zero)
+        container.addArrangedSubview(createButton(title: "Show side bar", action: #selector(presentSideBar)))
+    }
+
+    private struct Constants {
+        static let optionsSpacing: CGFloat = 5.0
+        static let labelSwitchSpacing: CGFloat = 10.0
+    }
+
+    private let sideBar: SideBar = {
+        return SideBar(frame: .zero);
+    }()
+
+    private let contentViewController: UIViewController = {
+        return UIViewController(nibName: nil, bundle: nil);
+    }()
+
+    @objc private func presentSideBar() {
+        contentViewController.modalPresentationStyle = .fullScreen
+        contentViewController.view.backgroundColor = view.backgroundColor
+
+        sideBar.insert(in: contentViewController.view)
+        sideBar.delegate = self
+
+        sideBar.topItems = [
+            TabBarItem(title: "Home", image: UIImage(named: "Home_28")!, selectedImage: UIImage(named: "Home_Selected_28")!, landscapeImage: UIImage(named: "Home_24")!, landscapeSelectedImage: UIImage(named: "Home_Selected_24")!),
```

Comment:
>Home_24 [](start = 157, length = 7)

We should not set the landscape image here. Side bar should always use the "portrait" image which is bigger. The naming is not ideal for ipad since tabbaritem and tabBarItemView were optimized for iPhone